src/components/ParkDetails/ParkDetails.js

```diff
@@ -6,31 +6,37 @@ import ParkActivities from '../ParkActivities/ParkActivities';
 import React, { useState, useEffect }from 'react';
 import SunriseSunset from '../SunriseSunset/SunriseSunset';
 
+import '../Error/Error.css'
+
 
 const ParkDetails = ({selectedPark, addToFutureTrips}) => {
 
   const [parkDetails, setParkDetails] = useState({});
+  const [error, setError] = useState('')
 
   const getDetails = () => {
     getParkDetailsData(selectedPark.parkCode)
       .then(data => {
         const details = filterParkDetails(data)
         setParkDetails(details)
       })
-  }
+      .catch(err => setError('Something went wrong. Please try again later!'))
+    }
 
   useEffect(() => {
     getDetails()
   }, [])
 
-    return(
+  return(
+    <div>
+    {(!Object.keys(parkDetails).length && error) ? <h2 className='error'>{error}</h2> :
       <main className='park-details'>
         <Link to='/parksByState'>
           <button>go back</button>
         </Link>
         {
-          !Object.keys(parkDetails).length ?
-          <h2>loading......</h2> :
+          (!Object.keys(parkDetails).length && !error) ?
+          <h2>Loading park details...</h2> :
```

Comment:
I mean, I think you really needed those three extra ... to really get the point across 😆